src/main/java/subway/LineFacade.java

```diff
@@ -0,0 +1,58 @@
+package subway;
+
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+@Component
```

Comment:
가독성을 위해 공백 라인을 추가해보면 좋을 것 같습니다.

```suggestion
import java.util.List;

@Component
```